packages/unity-interface/UnityInterface.ts

```diff
@@ -62,7 +62,33 @@ function resizeCanvas(targetHeight: number) {
   }
 }
 
+export function createUnityLogger(): ILogger {
```

Comment:
whats the diff between this fn and https://github.com/decentraland/kernel/blob/main/packages/shared/logger/index.ts#L29 ?

If the prefix its the only thing that changes, could we refactor that fn and use it for both cases ?

Reply:
The first prefix is `kernel`, and I changed it to `unity`.

I can modify the first function to get that. But I created a new function because if we want to modify something to report the logs from Unity, for example, not showing normal logs, or we want to send directly to rollbar and not show those in console, we can.

This will add A LOT of messages.

If you want, I can modify the first function to have a new parameter to change that `kernel` prefix. And when we need what I said, we create a new function like this one.